internal/relocated/relocated_bundle.go

```diff
@@ -0,0 +1,32 @@
+package relocated
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/docker/cnab-to-oci/relocation"
+	"github.com/docker/go/canonical/json"
+)
+
+type Bundle struct {
+	*bundle.Bundle
+	RelocationMap relocation.ImageRelocationMap
+}
+
+// FromBundle returns a RelocatedBundle with an empty relocation map.
+func FromBundle(bndl *bundle.Bundle) *Bundle {
+	return &Bundle{
+		Bundle:        bndl,
+		RelocationMap: relocation.ImageRelocationMap{},
+	}
+}
+
+// WriteRelocationMap serializes the relocation map and writes it to a file as JSON.
+func (b *Bundle) WriteRelocationMap(dest string, mode os.FileMode) error {
```

Comment:
I wonder if the code in the package `fetch` should be here instead, along with `WriteRelocationMap` ?

Reply:
hum IIRC it's not possible, the code has been extracted to avoid dependency cycle

Reply:
Oh.. that's too bad that read and write sections are not in the same package 😞

Reply:
Finally it looks like I'm able to put everything from `fetch` to `relocated`.
So that we have `relocated.FromBundle`, `relocated.BundleFromFile` but also `relocated.Bundle.Store`
I'll push soon the updated version